web/src/lib/components/shared-components/navigation-bar/navigation-bar.svelte

```diff
@@ -42,7 +42,7 @@
 
 <section
 	id="dashboard-navbar"
-	class="fixed w-screen  z-[100] bg-immich-bg dark:bg-immich-dark-bg text-sm"
+	class="fixed w-screen z-[900] bg-immich-bg dark:bg-immich-dark-bg text-sm"
```

Comment:
@alextran1502 Changing the z-index from 100 to 900 makes the modal backdrop no longer cover the navbar. Is that an intended change?

![image](https://user-images.githubusercontent.com/59014050/220374848-39b2fa46-0e1e-4d28-9afd-2c0f01798d56.png)

Reply:
No that is not, I will create a PR to fix this